client/src/components/Shows/NewShow/Form.js

```diff
@@ -88,15 +88,15 @@ class NewShowForm extends Component {
               )}
 
               <Field
-                className="grid-span--2"
+                className="grid-span--2, form-summary"
```

Comment:
@kmid5280 can you undo this and the `form-desc` class you're adding below? (That's what's changing the fields from being stacked to appearing left/right of each other, right?)